### PR TITLE
Tooltips: Clarify some option explanations

### PR DESCRIPTION
**Pull request by larsjohnsen**
Replaces pull request #199

### CODE REVIEW

File: src/UI/Settings/Notifications/Edit/NotificationEditViewTemplate.hbs [hunk 1]
```diff
@@ -33,7 +33,7 @@
                         </label>
 
                         <span class="help-inline-checkbox">
-                            <i class="icon-nd-form-info" title="Do you want to get notifications when episodes are grabbed?"/>
+                            <i class="icon-nd-form-info" title="Be notified when episodes are available for download and has been sent to a download client"/>
```
Comment (kayone) [inline src/UI/Settings/Notifications/Edit/NotificationEditViewTemplate.hbs]: I think the available part is implied. we can't send something that isn't available to the download clients. so I suggest,

"Be notified when episodes are sent to a download client"

Comment (larsjohnsen) [inline src/UI/Settings/Notifications/Edit/NotificationEditViewTemplate.hbs]: I agree. Feel free to modify if you merge.

**Comment by larsjohnsen** [comment]
Included in #214

File: src/UI/Settings/Notifications/Edit/NotificationEditViewTemplate.hbs [hunk 2]
```diff
@@ -77,7 +77,7 @@
                         </label>
 
                         <span class="help-inline-checkbox">
-                            <i class="icon-sonarr-form-info" title="Do you want to get notifications when episodes are upgraded to a better quality?"/>
+                            <i class="icon-sonarr-form-info" title="Be notifed when episodes are upgraded to a better quality"/>
```
Comment (kayone) [inline src/UI/Settings/Notifications/Edit/NotificationEditViewTemplate.hbs]: spelling in `notifed`

**Pull request closed without merging**